frontend/app/src/components/history/events/HistoryEventNote.vue

```diff
@@ -34,27 +36,32 @@ interface NoteFormat {
   amount?: BigNumber;
   asset?: string;
   url?: string;
+  chain?: Blockchain;
 }
 
-const formatNotes = (
-  notes: string,
-  amount: BigNumber | null,
-  assetId: string,
-  noTxHash: boolean
-): NoteFormat[] => {
+const {
+  notes,
+  amount,
+  asset: assetId,
+  noTxHash,
+  validatorIndex
+} = toRefs(props);
+
+const formatNotes: ComputedRef<NoteFormat[]> = computed(() => {
   const { assetSymbol } = useAssetInfoRetrieval();
 
-  const asset = get(assetSymbol(assetId));
+  const asset = get(assetSymbol(get(assetId)));
```

Comment:
`assetSymbol` gets `MaybeRef` right? Is there a problem with the typing? we should not have to do `get` in the parameter